keystoremanager/import_export_test.go

```diff
@@ -79,17 +82,21 @@ func TestImportExportZip(t *testing.T) {
 	passphraseByFile := make(map[string]string)
 
 	// Add non-root keys to the map. These should use the new passphrase
-	// because they were formerly unencrypted.
-	privKeyList := repo.KeyStoreManager.NonRootKeyStore().ListFiles(false)
+	// because the passwords were chosen by the newPassphraseRetriever.
```

Comment:
again, if someone doesn't choose a passphrase for targets, we still want to encrypt it at rest on export.